### ISSUE TITLE
Touch screen function lost on 1.5.13

### ISSUE THREAD

**Issue by Nickfly550** [FrSkyRC/ETHOS-Feedback-Community#4400]
On my X18 flying yesterday on 1.5.12 and all working fine Updated to 15.13 and lost touch screen everything else works on the radio.
Went back to 15.11 and touch screen worked as normal. Tried the update again, update worked as normal but when you power up the radio no touch screen function. Went flying today everything works using buttons and scroll wheel. Tried again and went back to 1.5.11 and touch screen worked straight away. loaded 1.5.13 again for the 3rd time and no touch function.

**Comment by rwijnhov** [comment]
yes, same issue here. Most of the time throttle not idle on x18 and x18s will trigger a non functioning touchscreen.  Rebooting the tx a couple of time with the throttle idle fixes it. Could also be it is the rebooting that fixes it.

**Comment by strgaltdel** [comment]
maybe this issue needs some more details.

i tried to reproduce without success
i'm on 1.5.13 x18s-eu firmware, x18se.

even with a failed throttle check (< -95%) on start i never have had touch problems.
tried five times in a row.
then changed to another model and tried another five times

maybe model template related ?

**Comment by rwijnhov** [comment]
No it's not a model template issue. Have this on different models. Try it again tomorrow Morning when it has been off for a longer time. It happenes to me 6 times now in 2 days ons both my x18 and x18s.

**Comment by bsongis-frsky** [comment]
@Nickfly550 @rwijnhov one of you could contact me on Discord? I need help to fix this issue, because I don't have the "regular" X18. Just one of the first X18 prototypes, which is in fact a X18S ...

**Comment by robthomson** [comment]
@bsongis-frsky ... you may need to provide a link to the discord group as they cant contact you unless on same group!

**Comment by bsongis-frsky** [comment]
Here:
https://discord.gg/zgwFrHyR

**Comment by Nickfly550** [comment]
@bsongis-frsky My radio is also a first release X18 that had the gyro added. I ordered 2 of them on pre release my friend who now has the other radio is having the same problem after the update.

**Comment by bsongis-frsky** [comment]
Would you please reach me on Discord. I have had a contact with somebody which doesn't have the issue so often. I would like somebody who is able to reproduce the problem almost 100%

**Comment by tars01** [comment]
I'm facing the same issues, upgraded to 1.5.13 last night on X18. I rebooted a few times and it went away. I have just powered it on again and the problem has returned.

**Comment by JHS-br1** [comment]
Hi, I have a X18 and it shows the same fault. Was running 1.5.11 upgraded to 1.5.13 through ethos suite. When complete turned off radio. Turned on looks OK, buttons work but touch screen does not accept inputs. Reinstalled 1.5.11 now working OK.

**Comment by FlipAirboost** [comment]
X18  ethos 1.5.13.eu #89915922  n° s 377a33753131

1. loss of touchscreen function for menu navigation but not for waking up from sleep mode! 
2. gyro fonction lost ( no choice in menu)

You should change the slogan "let set the limit" to "bozo software very limited" 😘

Regards Flip

**Comment by bsongis-frsky** [comment]
@JHS-br1 I have found a way to replicate the issue here. It should be good in the next nightlies I hope

@FlipAirboost If your radio really has a Gyro, then you just have to flash the X18S firmware and the issue is gone. @Nickfly550 already did it with success.

**Comment by rwijnhov** [comment]
@bsongis-frsky  you mean x18S bootloader + firmware right? Not only the firmware. Just to be sure.

**Comment by FlipAirboost** [comment]
> @FlipAirboost If your radio really has a Gyro, then you just have to flash the X18S firmware and the issue is gone. @Nickfly550 already did it with success.

OK, during the previous upgrade to ethos 1.4.x, the radio sometimes asked me, but not every time, for a calibration of the gyros. I haven't used this feature yet.

(I'm still using my x9d while waiting for a finalized version of ethos, something practical, for example for the inactivity alarm which triggers well after the programmed time but only once! on OpenTx you can set the trigger time and the repetition time ,less dangerous than the automatic radio shutdown requested by some people  #4100 )o: 

My radio is recognized as an x18 by ethos suite how do I make it recognized as an x18s?
![image](https://github.com/user-attachments/assets/867fc05f-0cac-41b6-a3f2-60814bfd8cbc)


The version of the gimbals is not the same on the x18s, does that have any influence?

**Comment by MIKE-DAILEY** [comment]
How to install the X18S bootloader and firmware on a X18 https://www.rcgroups.com/forums/showpost.php?p=52855293&postcount=2333

**Comment by FlipAirboost** [comment]
> How to install the X18S bootloader and firmware on a X18 https://www.rcgroups.com/forums/showpost.php?p=52855293&postcount=2333

Ok, I started to follow the procedure but I don't understand anything.
I'll wait for version 1.5.14.
It was still much simpler with Companion where you chose your radio yourself in the program.

**Comment by bsongis-frsky** [comment]
@FlipAirboost You may flash X18 or X18S firmware. They are the same now.

**Comment by JHS-br1** [comment]
Hi does that mean you have modified 1.5.13 to run on an early X18. Or 1.5.14 nightly.
Is the boot loader 1.4.15 the same for both X1r and X18S. Because I cannot find a boot loader for X18S only.
H

**Comment by FlipAirboost** [comment]
> @FlipAirboost You may flash X18 or X18S firmware. They are the same now.

@bsongis-frsky 
Hi Bertrand,

> X18 ethos 1.5.13.eu #89915922 n° s 377a33753131
> 
**> 1. loss of touchscreen function for menu navigation but not for waking up from sleep mode!**

I didn't change anything on my radio.
I was waiting for version 1.5.14 to eliminate the two aforementioned malfunctions.

Yesterday, I turned on the radio to present the software and menus to a colleague who was considering buying an x20 **and the touch screen was working again!!!**

However, I had turned the radio on and off 2 or 3 times before to check for the presence of the gyros.

Previously, I could no longer slide menus from one page to another, nor expand menus with the touch screen, nor select a main section.
The only thing that still worked in touch was exiting the screen from sleep mode.

Regards Phil
